.gitpod.yml

```diff
@@ -1,4 +1,5 @@
-image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:aledbf-trigger.4
+image:
+  file: dev/image/Dockerfile
```

Comment:
This move invites divergence between the environment we use for development and what we use for CI.
Please revert.

Reply:
It actually forces one of two things:
- It becomes someone's job to manually update the dev image, frequently and forever
- We constantly sink deeper into deprecation and continuously accumulate security vulnerabilities in our dev environnement

Reply:
> It becomes someone's job to manually update the dev image, frequently and forever

Exactly - otherwise we succumb to undefined versions and configuration drift.

> Otherwise we constantly sink deeper into deprecation and continuously accumulate disclosed security vulnerabilities in our dev environnement

I don't see how that's different with using a `Dockerfile` in the `.gitpod.yml`, except that we then have at least two places that we need to keep up to date: the `Dockerfile` and the image we use in CI and elsewhere.